DataPipeline-Integration-Orc-Upgrade-Libraries/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id 'java'
+    id 'eclipse'
+}
+
+repositories {
+    //mavenLocal()
+    jcenter()
+    maven {
+        url = 'https://maven.twttr.com/'
+    }
+    maven {
+        url = 'https://jitpack.io'
+    }
+    maven {
+        url = 'https://maven.northconcepts.com/public/repositories/datapipeline'
```

Comment:
Let's add the `content` filter for NC as @chlocarlos recently did in TemplateMaster.

Reply:
Done.